scripts/avalanchego-installer.sh

```diff
@@ -366,14 +366,29 @@ else
   ipChoice="2"
 fi
 echo ""
-echo "Your node can accept RPC calls on port 9650. If restricted to local, it will be accessible only from this machine."
-while [ "$rpcOpt" != "any" ] && [ "$rpcOpt" != "local" ]
+echo "Your node accepts RPC calls on port 9650. If restricted to private, it will be accessible only from this machine."
+echo "Only p2p port (9651 by default) NEEDS to be publicly accessible for correct node operation, RPC port is used for"
+echo "interaction with the node by the operator or applications and SHOULD NOT be freely accessible to the public."
+echo ""
+while [ "$rpcOpt" != "public" ] && [ "$rpcOpt" != "private" ]
 do
-  read -p "Do you want the RPC port to be accessible to any or only local network interface? [any, local]: " rpcOpt
+  read -p "Do you want the RPC port to be accessible to public or private (local) network interface? [public, private]: " rpcOpt
```

Comment:
We should explicitly add here that the recommended for a validator is private.

Reply:
It is mentioned in the confirmation text, but I'll pull it up to here.